bors.toml

```diff
@@ -0,0 +1,4 @@
+status = ["Snap"]
+delete-merged-branches = true
+block-labels = [ "no-merge" ]
+timeout-sec = 1200
```

Comment:
This build takes a long time (over 30min), so is this timeout enough?

Reply:
Apparently so :shrug:…